packages/webdriverio/src/commands/element/setValue.ts

```diff
@@ -1,5 +1,7 @@
 import type { AddValueOptions } from './addValue'
 
+type Value = string | number
```

Comment:
Let's import this from `addValue`
```suggestion
import type { AddValueOptions, ValueType } from './addValue'
```